contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -39,8 +39,24 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
   // |                       Initialization                         |
   // ================================================================
 
-  constructor(TermsOfServiceAllowListConfig memory config) ConfirmedOwner(msg.sender) {
+  constructor(
+    TermsOfServiceAllowListConfig memory config,
+    address[] memory initialAllowedSenders,
```

Comment:
Are we going to run into gas issues if the allowlist gets too large?

Also, what will the EngOps process be for this contract upgrade?

Reply:
I wonder if a "admin add" feature is a better option where the contract owner can manually add people to the allowlist.

Reply:
Should be able to just automate all of the process with tooling. 

As for size, I was thinking that too. Let me add a unit test to see how many we can reasonably add. I agree that another method will be needed if we need to batch add people.

Reply:
Repeating from slack thread: 
Tested up to 500k addresses via a unit test.
Leaving the unit test out because it does slow down the test suite by 20-30 seconds.